packages/@o3r/core/src/core/interfaces/item-identifier.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Identifier of an item
```

Comment:
I think the name and the description are too much generic.
The interface is actually used to identify components and configs in metadata (more generally in JSONs as it is also the case for Rules Engine's rules which are not metadata).

I would suggest to prefix the interface with `Otter` (or `o3r`) and give more details on the interface and fields' descriptions

Reply:
I would not prefix all our interfaces with Otter as it already comes from an o3r package, however, if you want a prefix maybe `Metadata` would make sense?